streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -2894,40 +2909,33 @@ public void shouldSupportTriggerMaterializedWithKTableFromKStream() {
             .to(output);
 
         final Topology topology = builder.build(props);
-        final String topologyDescription = topology.describe().toString();
 
+        final String topologyDescription = topology.describe().toString();
         assertThat(
             topologyDescription,
             equalTo("Topologies:\n" +
                 "   Sub-topology: 0\n" +
                 "    Source: KSTREAM-SOURCE-0000000000 (topics: [input])\n" +
-                "      --> KSTREAM-TOTABLE-0000000002\n" +
-                "    Processor: KSTREAM-TOTABLE-0000000002 (stores: [])\n" +
+                "      --> KSTREAM-TOTABLE-0000000001\n" +
+                "    Processor: KSTREAM-TOTABLE-0000000001 (stores: [])\n" +
                 "      --> KTABLE-MAPVALUES-0000000003\n" +
                 "      <-- KSTREAM-SOURCE-0000000000\n" +
                 "    Processor: KTABLE-MAPVALUES-0000000003 (stores: [store])\n" +
                 "      --> KTABLE-TOSTREAM-0000000004\n" +
-                "      <-- KSTREAM-TOTABLE-0000000002\n" +
+                "      <-- KSTREAM-TOTABLE-0000000001\n" +
                 "    Processor: KTABLE-TOSTREAM-0000000004 (stores: [])\n" +
                 "      --> KSTREAM-SINK-0000000005\n" +
                 "      <-- KTABLE-MAPVALUES-0000000003\n" +
                 "    Sink: KSTREAM-SINK-0000000005 (topic: output)\n" +
                 "      <-- KTABLE-TOSTREAM-0000000004\n\n"));
 
         try (
-            final TopologyTestDriver driver = new TopologyTestDriver(
-                topology,
-                mkProperties(mkMap(
-                    mkEntry(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy"),
-                    mkEntry(StreamsConfig.APPLICATION_ID_CONFIG, "test"),
-                    mkEntry(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory("kafka-test").getAbsolutePath())
-                )),
-                Instant.ofEpochMilli(0L))) {
+            final TopologyTestDriver driver = new TopologyTestDriver(topology, props)) {
```

Comment:
Just a code simplification -- no need to not use `props` similarly to the other tests.